DuckDuckGo/AuthenticationViewController.swift

```diff
@@ -58,18 +58,37 @@ class AuthenticationViewController: UIViewController {
             onCouldNotAuthenticate()
         }
     }
+    
+    class Authenticator {
+        func canAuthenticate() -> Bool {
+            return true
+        }
+
+        func authenticate(reason: String) async throws -> Bool {
+            
+            return true
+        }
```

Comment:
This is inaccurate in a couple ways:

1. The function is `async` but it's not `await`ing anything
2. It returns true no matter what, when instead it should be going through `LocalAuthentication` - in this state, the authentication feature will never present an authentication prompt